app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,137 @@
+      <p>You'll teach computer science as a GCSE qualification. You'll teach pupils how to develop their capability, creativity and knowledge in computing. </p>
+      <ul class="vertical-tags">
+        <li class="tag">Computational thinking</li>
+        <li class="tag">Digital media and information technology  </li>
+        <li class="tag">Computer networks </li>
+        <li class="tag">Cyber security  </li>
+        <li class="tag">Data representation </li>
+        <li class="tag">Digital technology   </li>
+      </ul>
+      <p>For 16 to 18 year olds (key stage 5), you'll teach computer science as a A Level qualification and encourage pupils to develop a broad range of the knowledge and skills, preparing them for further learning or employment, with themes like:  </p>
+      <ul class="vertical-tags">
+        <li class="tag">Abstraction   </li>
+        <li class="tag">Writing programmes to solve problems </li>
+        <li class="tag">Efficiency of algorithms </li>
+        <li class="tag">Data structures and the use of databases</li>
+        <li class="tag">Systems architecture </li>
+        <li class="tag">Computing related mathematics  </li>
+      </ul>
+
+
+    <p><a href="https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study/national-curriculum-in-england-computing-programmes-of-study">Read the full national curriculum for more information about teaching computing in secondary schools</a>.</p>
+     </div>
+</section>
+
+<section class="col col-845 inset">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/images/content/subjects/rotna-roy.jpeg",
+          heading: "Empowering my students",
+          text: "quote",
+          accreditation: "Rotna Roy,",
+        }
+      ],
+      colors: %w[yellow],
+      numbered: false
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h2 class="heading-s">Funding for training to teach computing </h2>
+
+    <p>To train to be a computing teacher, you'll need to do a teacher training course. </p>
+
+<p>Bursaries of £28,000 and scholarships of £30,000 are available for <a href="/funding-and-support/scholarships-and-bursaries">eligible trainee computing teachers</a>. </p>
+
+ <p>You can receive this alongside a tuition fee and maintenance loan. </p>
+
+  <p>You may also be able to get extra funding support if you are: </p>
+    <ul>
+      <li>a parent or carer </li>
+      <li>disabled </li>
+    </ul>
+<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding. </p>
+
+
+    <h2 class="heading-s">Check your qualifications</h2>
+
+<p>You'll need a bachelors degree before training to teach but it does not have to be in computing. Alongside computing, The most popular degrees people have before doing a teacher training course in computing are:</p>
```

Comment:
Alongside computing, **The** most popular degrees people have before doing a teacher training course in computing are

@caitlin-reidy Could you switch this sentence around like below? If not typo in the as it should be lower case.

The most popular degrees apart from computing that people have before doing a teacher training course in computing are

Reply:
@caitlin-reidy it would be good if we could also give some examples of the more unusual degrees people have done that have gone on to do a teacher training course in computing. Otherwise people may just think they can only do it if they've got a degree in a computer related subject and it may put them off applying?